docs/02-get-started/02-deploy-expose-function.md

```diff
@@ -0,0 +1,158 @@
+
+## Create a function
+
+First, let's create the Function and apply it.
+
+<div tabs name="Deploy a Function" group="deploy-expose-function">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+kyma init function --name hello-world
+kyma apply function
+```
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. In Kyma Dashboard, go to the `default` Namespace.
+2. Go to **Workloads** > **Functions**.
+3. Click on **Create Function +**.
+4. Name the Function `hello-world` and click **Create**.
+  </details>
+</div>
+
+
+### Verify the Function deployment
+
+Now let's make sure that the Function has been deployed successfully. 
+
+<div tabs name="Verify the Function deployment" group="deploy-expose-function">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+kubectl get functions hello-world 
+```
+
+The operation was successful if the statuses for **CONFIGURED**, **BUILT**, and **RUNNING** are `True`.
+
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+The operation was successful if the Function **Status** changed from `DEPLOYING` to `RUNNING`.
+
+> **NOTE:** You might need to wait a few seconds for the status to change.
+  </details>
+</div>
+
+## Expose the Function
+
+After we've got our `hello-world` Function deployed, we might want to expose it outside our cluster so that it's available for other external services.
+
+> **CAUTION:** Exposing a workload to the outside world is always a potential security vulnerability, so tread carefully. In a production environment, always [secure the workload](../03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md) you expose.
+
+First, let's create an [APIRule](../05-technical-reference/00-custom-resources/apix-01-apirule.md) for the Function.
+
+<div tabs name="Expose the Function" group="deploy-expose-function">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+cat <<EOF | kubectl apply -f -
+  apiVersion: gateway.kyma-project.io/v1alpha1
+  kind: APIRule
+  metadata:
+    name: hello-world
+    namespace: default
+  spec:
+    gateway: kyma-gateway.kyma-system.svc.cluster.local
+    rules:
+      - accessStrategies:
+        - config: {}
+          handler: allow
+        methods:
+          - GET
+          - POST
+          - PUT
+          - PATCH
+          - DELETE
+          - HEAD
+        path: /.*
+    service:
+      host: hello-world.$CLUSTER_DOMAIN
+      name: hello-world
+      port: 80
+EOF
+```
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. In your Function's view, go to the **Configuration** tab.
+2. Click on **Expose Function +**.
+3. Provide the **Name** (`hello-world`) and **Subdomain** (`hello-world`) and click **Create**.
```

Comment:
No field is called **Subdomain** :'(

![image](https://user-images.githubusercontent.com/76950046/133642291-93c38f70-d265-4dff-b99a-3acd8349c868.png)

Reply:
I think you don't have the latest Busola image. 
Last time I checked, the `latest` flag didn't work and you needed to replace that with the latest commit. 
But I'll double-check.

Reply:
That's what's in the latest Busola image (`104f0517`): 
![image](https://user-images.githubusercontent.com/53601578/133767877-24b4facb-ce47-4612-bd79-2ace60f8aa2b.png)